README.md

```diff
@@ -325,7 +318,7 @@ The plugin automatically shuts down the HikariCP connection pool when the ActorS
 This is done using [ActorSystem.registerOnTermination](https://doc.akka.io/api/akka/current/akka/actor/ActorSystem.html#registerOnTermination[T](code:=>T):Unit).
 
 [slick]: http://slick.lightbend.com/
-[slick-jndi]: http://slick.typesafe.com/doc/3.1.1/database.html#using-a-jndi-name
+[slick-jndi]: http://slick.typesafe.com/doc/3.2.3/database.html#using-a-jndi-name
```

Comment:
You bit me on this. I was about to send a PR for updating it to 3.3.0.

Slick 3.3.0 is out. I did the release today and still working on updating the website.

Reply:
Anyway, I will send a PR with the upgrade ASAP. I would like to have it included on this next release. There is some important fixes and most important upgrade of Hikari.